include/zephyr/posix/posix_features.h

```diff
@@ -331,35 +322,50 @@ __PICOLIBC_MINOR__ == 8 && __PICOLIBC_PATCHLEVEL__ >= 9)))
 #define NZERO      (20)
 
 /* Runtime invariant values */
-#define AIO_LISTIO_MAX     _POSIX_AIO_LISTIO_MAX
-#define AIO_MAX            _POSIX_AIO_MAX
-#define AIO_PRIO_DELTA_MAX (0)
-#define DELAYTIMER_MAX     _POSIX_DELAYTIMER_MAX
-#define HOST_NAME_MAX      _POSIX_HOST_NAME_MAX
-#define LOGIN_NAME_MAX     _POSIX_LOGIN_NAME_MAX
-#define MQ_OPEN_MAX        _POSIX_MQ_OPEN_MAX
-#define MQ_PRIO_MAX        _POSIX_MQ_PRIO_MAX
-
+#define AIO_LISTIO_MAX                _POSIX_AIO_LISTIO_MAX
+#define AIO_MAX                       _POSIX_AIO_MAX
+#define AIO_PRIO_DELTA_MAX            (0)
+#ifndef ARG_MAX
+#define ARG_MAX                       _POSIX_ARG_MAX
+#endif
 #ifndef ATEXIT_MAX
-#define ATEXIT_MAX 8
+#define ATEXIT_MAX                    (32)
+#endif
+#define DELAYTIMER_MAX \
+	COND_CODE_1(CONFIG_POSIX_TIMERS, (CONFIG_POSIX_DELAYTIMER_MAX), (0))
+#define HOST_NAME_MAX \
+	COND_CODE_1(CONFIG_POSIX_NETWORKING, (CONFIG_POSIX_HOST_NAME_MAX), (0))
```

Comment:
How is this related to `_POSIX_HOST_NAME_MAX`, is `_POSIX_HOST_NAME_MAX` set as a maximum allowed value, and this `HOST_NAME_MAX` it the actual configured max value?